src/widgets/component-new-edit/components/response-format/simple/simple-numeric.jsx

```diff
@@ -44,14 +58,42 @@ function ResponseFormatDatatypeNumeric({ name, required, readOnly }) {
           label={Dictionary.decimals}
           disabled={readOnly}
         />
-        <SelectMetaDataContainer
-          type="units"
-          name="unit"
-          label={Dictionary.unit}
-          emptyValue={Dictionary.unitEmptySelect}
-          mapMetadataFunction={mapUnitData}
+        <Field
+          name="isDynamicUnit"
+          component={ListRadios}
+          label={Dictionary.dynamicUnit}
           disabled={readOnly}
-        />
+          onChange={handleDynamicUnitChange}
+          required
+          // Convert string "true"/"false" to boolean true/false when storing in Redux form
+          parse={value => value === 'true'}
+          // Convert boolean true/false to string "true"/"false" when displaying the form
+          format={value => (value === true ? 'true' : 'false')}
+        >
+          <GenericOption key="1" value="true">
```

Comment:
If the `key` is meant as an React `key` to identify an item in a list, it is better practice to use some sort of `id` instead of an index (eg `"dictionary-true"` instead of `"1"`) since the `key` id should never change for each item.

See: https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key

Here it doesn't seem to be useful since it's not an array though so we should remove this property since `key` does not seem to be used by the `GenericOption` component.